src/fsharp/xlf/FSComp.txt.cs.xlf

```diff
@@ -83,18 +83,18 @@
         <note />
       </trans-unit>
       <trans-unit id="listElementHasWrongType">
-        <source>All elements of a list constructor expression must have the same type. This expression was expected to have type '{0}', but here has type '{1}'.</source>
-        <target state="translated">Všechny elementy výrazu konstruktoru seznamu musí mít stejný typ. Očekávalo se, že tento výraz bude mít typ {0}, ale tady je typu {1}.</target>
+        <source>All elements of a list constructor expression must have the same type. The first element has type '{0}', but there is also an element of type '{1}'.</source>
```

Comment:
@cartermp ideally, this should be handled outside this repository with separate integration pipeline.

This repository would pull those files from the localization repository during the build and run the command to update any missing entry before compiling.

The localization repository would have an automated integration task pulling the fscomp changes in master from here and check them in the localization repository.

The case a contributor to Microsoft/visualfsharp is also going to handle the localization is very slim (less than 1%?) and we have to deal with increased pipeline / feedback loop anytime someone works on better errors (just dealing with the test suite is enough), this is a recurrent contribution pattern that we want to streamline.

Reply:
I'll add that the noise generated in code reviews and additional checkins increase potential errors and discourage focused reviews.